src/components/CacheDebugPanel.tsx

```
@@ -0,0 +1,153 @@
+
+  // Only show in development unless explicitly enabled for production
+  if (!showInProduction && process.env.NODE_ENV === 'production') {
+    return null;
+  }
+
+  const refreshStats = () => {
+    setStats(cacheStorage.getStats());
+    setStorageInfo(localStorageUtils.getInfo());
+  };
+
+  useEffect(() => {
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)